crypto/thread/build.info

```diff
@@ -1,16 +1,16 @@
 LIBS=../../libcrypto
 
-IF[{- !$disabled{'thread-pool'} -}]
-  $THREADS_ARCH=\
-        arch.c  \
-        arch/thread_win.c arch/thread_posix.c arch/thread_none.c
+$THREADS_ARCH=\
+      arch.c  \
+      arch/thread_win.c arch/thread_posix.c arch/thread_none.c
+
+IF[{- !$disabled{quic} -}]
+  SOURCE[../../libssl]=$THREADS_ARCH
```

Comment:
Why not just SHARED_SOURCE anymore?

Reply:
Because the reason for SHARED_SOURCE is to avoid duplicates in a no-shared build. But in a no-thread-pool no-shared we end up with no definitions in libssl.

But this still isn't quite right....another update coming...